apps/nrm/behaviours/types-of-exploitation.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+
+  /**
+   * Check if at least one address field has been populated
+   * @param {*} req - request object
+   */
+  // eslint-disable-next-line complexity
+  isAtLeastOneOptionSelected(req) {
+    if (
+      req.form.values['types-of-exploitation-forced-to-work'] ||
+      req.form.values['types-of-exploitation-wages-taken'] ||
+      req.form.values['types-of-exploitation-forced-to-commit-fraud'] ||
+      req.form.values['types-of-exploitation-prostitution'] ||
+      req.form.values['types-of-exploitation-child-exploitation'] ||
+      req.form.values['types-of-exploitation-taken-somewhere'] ||
+      req.form.values['types-of-exploitation-forced-to-commit-crime'] ||
+      req.form.values['types-of-exploitation-organs-removed'] ||
+      req.form.values['types-of-exploitation-unpaid-household-work'] ||
+      req.form.values['types-of-exploitation-other']
```

Comment:
would it be worth checking if key in `req.form.values` starts with the value `types-of-exploitation`

something like
https://stackoverflow.com/questions/27507110/check-if-a-javascript-object-has-a-property-name-that-starts-with-a-specific-str

Reply:
Done.